test/backend_test_util.py

```diff
@@ -7,7 +7,7 @@
     from unittest.mock import DEFAULT, Mock
 except ImportError:
     # See https://github.com/python/mypy/issues/1153#issuecomment-253842414
-    from mock import DEFAULT, Mock  # type: ignore
+    from unittest.mock import DEFAULT, Mock  # type: ignore
```

Comment:
probably shouldn't allow changes like this (or just eliminate this try-except altogether if we don't want to support py2)

Reply:
the goal is to remove any vestiges from python2 days, I'll clean this up